mlir-tensorrt/compiler/include/mlir-tensorrt-c/Runtime/Runtime.h

```diff
@@ -221,10 +221,15 @@ mtrtRuntimeClientGetNumDevices(MTRT_RuntimeClient client, int32_t *numDevices);
 MLIR_CAPI_EXPORTED MTRT_Status mtrtRuntimeClientGetDevice(
     MTRT_RuntimeClient client, int32_t index, MTRT_Device *device);
 
-/// Retrieve the runtiem client that manages the specified memref.
+/// Retrieve the runtime client that manages the specified memref.
 MLIR_CAPI_EXPORTED MTRT_RuntimeClient
 mtrtMemRefGetClient(MTRT_MemRefValue memref);
 
+/// Retrieve the runtime client allocated cpu and gpu memory.
```

Comment:
This isn't quite accurate. You're reporting the CPU/GPU memory that is being tracked by the RuntimeClient. It can track buffers that are externally allocated.